README.md

```diff
@@ -49,3 +49,15 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
 ```php
 ./vendor/bin/psalm
 ```
+
+### Support the project
+
+[![Open Collective](https://img.shields.io/badge/Open%20Collective-sponsor-7eadf1?logo=open%20collective&logoColor=7eadf1&labelColor=555555)](https://opencollective.com/yiisoft)
+
+### Follow update
+
+[![Yii2](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](https://www.yiiframework.com/)
+[![Twitter Follow](https://img.shields.io/badge/twitter-follow-1DA1F2?logo=twitter&logoColor=1DA1F2&labelColor=555555?style=flat)](https://twitter.com/yiiframework)
+[![Telegram](https://img.shields.io/badge/telegram-join-1DA1F2?style=flat&logo=telegram)](https://t.me/yii3ru)
+[![Facebook](https://img.shields.io/badge/facebook-join-1DA1F2?style=flat&logo=facebook&logoColor=ffffff)](https://www.facebook.com/groups/yiitalk)
+[![Slack](https://img.shields.io/badge/slack-join-1DA1F2?style=flat&logo=slack)](https://join.slack.com/t/yii/shared_invite/enQtMzQ4MDExMDcyNTk2LTc0NDQ2ZTZhNjkzZDgwYjE4YjZlNGQxZjFmZDBjZTU3NjViMDE4ZTMxNDRkZjVlNmM1ZTA1ODVmZGUwY2U3NDA)
```

Comment:
That is potentially a problem. Sometimes slack resets the token so it changes. If we'll list it here, we'll have to update it everywhere.

Reply:
no direct link to slack?

Reply:
Nope. We can likely create a redirect-URL at yiiframework.com...

Reply:
I think we can link to https://www.yiiframework.com/chat

Reply:
Just place https://yii.slack.com/archives/

Reply:
@xepozz won't work. It's not an invitation link.

Reply:
Now there's way to have a good link. Here it is: https://yiiframework.com/go/slack